view-in-grafana.sh

```diff
@@ -45,7 +45,7 @@ done
 echo "ready"
 
 echo "Extracting data from tarballs..."
-find "$datadir" -type f -ctime -$RETENTION_DAYS -iname "*.bz2" -exec bash -c 'tar jxf "{}" -C $(dirname "{}")' \; 2>/dev/null;
+find "$datadir" -type f -ctime -$RETENTION_DAYS -iname "*.gz" -exec bash -c 'tar xf "{}" -C $(dirname "{}")' \; 2>/dev/null;
```

Comment:
We probably shouldn't replace this though right?  We should just unzip files in either format so we should leave the bz2 line and add the gz line?